src/frontend/js/index.tsx

```diff
@@ -54,12 +54,12 @@ document.addEventListener('DOMContentLoaded', event => {
         }
 
         try {
-          addLocaleData(require(`react-intl/locale-data/${localeCode}`));
```

Comment:
IMO `index` is not the right place to do this: it is impossible for React code to reach this code and load another locale than the one that was loaded at start. At the same time we're not aggregating locale loads between render trees because this is in the iterator.

The problems are theoretical as we don't allow locale switching right now but we should be aware of the possible implications:
- load in index (but outside the iterator) to avoid multiple loads of locale data
- load inside a component (`App?`) so language can be changed and locale data dynamically reloaded when this happens